index.html

```diff
@@ -2829,10 +2829,66 @@
           <li>
             <a>Queue a task</a> on the <a>user interaction task source</a> to
             run the following steps:
-            <ol>
+            <ol data-link-for="PaymentAddress">
+              <li>Let <var>address</var> be a new instance of
+              <a>PaymentAddress</a>.
+              </li>
+              <li>Set the <a>addressLine</a> attribute of <var>address</var> to
```

Comment:
You cannot set attributes that are readonly. There need to be internal slots for all of these, and then the attributes need to have normative text that their getters return the value of the internal slot.

This only slipped by until now because this whole thing was underspecified. Thanks for catching it :)

Reply:
Good point. I'll add the slots.